src/ui-misc.cc

```diff
@@ -1479,33 +1481,7 @@ GList* get_action_items()
 					action_item->name = g_strdup(action_name);
```

Comment:
`action_name`, `label`, `tooltip` leak now.
```suggestion
					action_item->name = action_name;
```
Also for `action_item->label`.